integration/explain_compat_test.go

```diff
@@ -67,28 +67,28 @@ func testExplainCompatError(t *testing.T, testCases map[string]explainCompatTest
 			t.Run(targetCollection.Name(), func(t *testing.T) {
 				t.Helper()
 
-				explainParams := bson.D{
-					{tc.command, targetCollection.Name()},
-				}
+				explainTarget := bson.D{{tc.command, targetCollection.Name()}}
+				explainCompat := bson.D{{tc.command, compatCollection.Name()}}
```

Comment:
I noticed that both target and compat were using `targetCollection`. compat should use compatCollection, target should use targetCollection.